HOWTOCONTRIBUTE.rst

```diff
@@ -125,7 +125,7 @@ Preparing Pull Requests
 
 #. Install dependencies into a new conda environment::
 
-    $ conda env update -f ci/requirements.yml
+    $ conda env update -f ci/dev.yml
```

Comment:
Does ci/dev.yml exist? Couldn't see it here?

Reply:
Found it it's in the root dir.
https://github.com/xarray-contrib/xskillscore/blob/add_github_actions/dev.yml
Should the .yml and .txt be put in the ci folder or they need to be in the root dir for GitHub actions?

Reply:
They can stay in the `ci` folder for organization. In the `.github/workflows` folder I point to these different yml files where necessary. I changed it to `dev.yml` since it's really just the most extensive environment we have and only necessary for developers.